### ISSUE TITLE
Change name of `asel` in timeseries to something more comprehensible.

### ISSUE THREAD

**Issue by hmacdope** [MDAnalysis/mdanalysis#3911]
I know we're just doing the same as what memoryreader does, but it just struck me that `asel` is such a weird parameter name for an atomgroup. Not much we can do here, but if I'm not the only one maybe we can consider changing this for 3.0 (in a different PR, let's get this merged!).

_Originally posted by @IAlibay in https://github.com/MDAnalysis/mdanalysis/pull/3890#discussion_r1014879026_
     

I agree with @IAlibay that the name `asel` is a little confusing and could do with a change in 3.0. 
leave your opinions below. 

Will require a deprecation warning first if we go ahead.

**Comment by orbeckst** [comment]
yes, please, ~~`asel` ▶️  `selection` ?~~ EDIT: atomgroup (see below)

**Comment by IAlibay** [comment]
> yes, please, `asel` ▶️ `selection` ?

`asel` is an atomgroup right? `atomgroup` might be closer to what we do elsewhere?

**Comment by orbeckst** [comment]
yeah, you're right

**Comment by hmacdope** [comment]
@MDAnalysis/gsoc-mentors making this a gsoc-starter issue

**Comment by IAlibay** [comment]
@hmacdope isn't this a 3.0 targeted change? We wouldn't be able to merge it during the application period.

**Comment by hmacdope** [comment]
Ah yes sorry I meant the deprecation warning part. Do we think that is suitable ?

**Comment by IAlibay** [comment]
> Ah yes sorry I meant the deprecation warning part. Do we think that is suitable ? 

Sounds good to me!

**Comment by tushdemort** [comment]
hi @hmacdope, I want to work on MDAnalysis as part of GSoC and wanted to take up this issue. Seems like it won't get merged during the application period. However, can I work on it?

**Comment by hmacdope** [comment]
Please go ahead and make a PR! If you make a fix it can be merged in the application period.

**Comment by tushdemort** [comment]
Hi @hmacdope

I have made changes for **asel** in timeseries wherever the function is defined and called. I have also made changes in the test code for the same. Do you think **sample_atom_group** is a good variable name in place of asel (can be a little wordy)? atom_group and atomgroup are used as variable/function names in a couple of files.

Also, I didn't understand the discussion regarding the deprecation warning, can you explain it a bit?

**Comment by IAlibay** [comment]
@hmacdope I'm reopening to keep an issue to track the removal of asel - please do reclose if there's another one already.

**Comment by hmacdope** [comment]
Cheers, nah this is it.
